package.json

```diff
@@ -256,6 +266,11 @@
         "command": "one.editor.cfg.setDefaultValues",
         "title": "ONE: Set Default Values",
         "category": "ONE"
+      },
+      {
+        "command": "one.editor.mpqEditor.showFromDefaultExplorer",
```

Comment:
```suggestion
        "command": "one.editor.mpq.showFromDefaultExplorer",
```

How about this command name? 'editor' is duplicate. :-D

Reply:
Sorry. Ok.